src/components/AdditionalContacts/index.tsx

```diff
@@ -0,0 +1,301 @@
+        <Label className="text-normal" htmlFor="systemIntakeContact.role">
+          {t('contactDetails.additionalContacts.role')}
+        </Label>
+        <FieldErrorMsg>{errors.role}</FieldErrorMsg>
+        <Dropdown
+          id="IntakeForm-ContactRole"
+          name="systemIntakeContact.role"
+          value={activeContact.role}
+          onChange={e =>
+            setActiveContact({ ...activeContact, role: e.target.value })
+          }
+        >
+          <option value="" disabled>
+            {t('contactDetails.additionalContacts.select')}
+          </option>
+          {contactRoleOptions.map(option => (
+            <option key={option} value={option}>
+              {t(option)}
+            </option>
+          ))}
+        </Dropdown>
+      </FieldGroup>
+
+      {/* Action Buttons */}
+      <div className="margin-top-2">
+        <Button type="button" outline onClick={() => setActiveContact(null)}>
+          Cancel
+        </Button>
+        <Button type="button" onClick={() => handleSubmit()}>
+          {t(
+            activeContact?.id
+              ? 'contactDetails.additionalContacts.save'
+              : 'contactDetails.additionalContacts.addContact'
+          )}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+export default function AdditionalContacts({
+  systemIntakeId,
+  activeContact,
+  setActiveContact,
+  contacts,
+  createContact,
+  updateContact,
+  deleteContact
+}: {
+  systemIntakeId: string;
+  activeContact: SystemIntakeContactProps | null;
+  setActiveContact: (contact: SystemIntakeContactProps | null) => void;
+  contacts: SystemIntakeContactProps[] | [];
+  createContact: (
+    contact: SystemIntakeContactProps,
+    callback?: () => any
+  ) => void;
+  updateContact: (
+    contact: SystemIntakeContactProps,
+    callback?: () => any
+  ) => void;
+  deleteContact: (id: string, callback?: () => any) => void;
+}) {
+  const { t } = useTranslation('intake');
+
+  return (
+    <div className="system-intake-contacts margin-top-4">
+      {contacts.length > 0 && (
+        <>
+          <h4>{t('contactDetails.additionalContacts.title')}</h4>
+          <div className="system-intake-contacts__contacts-list">
+            {contacts.map(contact => {
+              // Show form if editing contact
+              if (activeContact && activeContact?.id === contact.id) {
+                return (
+                  <div key={contact.euaUserId}>
+                    <ContactForm
+                      key={contact.euaUserId}
+                      activeContact={activeContact}
+                      setActiveContact={setActiveContact}
+                      onSubmit={updateContact}
+                    />
+                    <Button
+                      type="button"
+                      unstyled
+                      className="text-error margin-top-2"
+                      onClick={() =>
+                        deleteContact(activeContact.id as string, () =>
```

Comment:
Small Typescript FYI - you can simplify this assertion to `activeContact.id!`, which asserts that a value isn't null or undefined. ([Docs](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-))